### PR TITLE
Add additional logging provider documentation 

### PR DESCRIPTION
**Pull request by noccor**
Adds documentation for Heroku, AWS Kinesis/Cloudwatch & HTTP.

### CODE REVIEW

**Review by matsimitsu — APPROVED** [review]

File: source/posts/logging/platforms/http.mdx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+#### attributes
+
+Attributes that can be used to more easily filter or search the message, with the following rules:
+
+- Attributes cannot be nested, or contain arrays
+- The following primitives are accepted as values:
+  - string ideally used as “tags” e.g. one-word values
+  - bool (true/false)
+  - int Integer numbers (10)
```
Comment (roytomeij) [inline source/posts/logging/platforms/http.mdx:149]:
```suggestion
  - int Integer numbers (`10`)
```

File: source/posts/logging/platforms/http.mdx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  - double Double/Floats (0.01)
```
Comment (roytomeij) [inline source/posts/logging/platforms/http.mdx:150]:
```suggestion
  - double Double/Floats (`0.01`)
```

File: source/posts/logging/platforms/http.mdx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+  ⚠️ Your log data must not contain any personal data, such as names, email
+  addresses, etc. It is your responsibility to ensure this data is filtered out
+  before being sent to AppSignal, and when identifying a person is necessary
+  that your application uses alternative forms of identification such as a user
+  ID, hash, or pseudonym.
+</Notice>
+
+## Supported Formats
+
+You can `POST` logs to AppSignal in several formats, more detailed information about each format can be found in the [Log Formatting documentation](/logging/formatting.html), via the links below.
```
Comment (roytomeij) [inline source/posts/logging/platforms/http.mdx:24]:
```suggestion
You can `POST` logs to AppSignal in several formats. More detailed information about each format can be found in the [Log Formatting documentation](/logging/formatting.html), via the links below.
```

File: source/posts/logging/platforms/http.mdx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+- [JSON](/logging/formatting.html#json)
+- [NDJSON](/logging/formatting.html#ndjson)
+- [Logfmt](/logging/formatting.html#logfmt)
+- [Plaintext](/logging/formatting.html#plaintext)
+
+**Please note:** [`NDJSON`](#http-configuraiton-for-ndjson) and [`Syslog`](#http-configuration-for-syslog) use a different endpoint than `JSON` and `Logfmt` logs.
+
+<Notice variant="warning">
+  AppSignal will silently drop any message that does not contain the expected
+  format, fields or incorrect values.
```
Comment (roytomeij) [inline source/posts/logging/platforms/http.mdx:35]:
```suggestion
  format, fields, or incorrect values.
```

**Review by roytomeij — CHANGES_REQUESTED** [review]
In addition to my line comments, I think we could also use the [Redirector](https://github.com/appsignal/handbook/blob/main/support/redirector.md) to include links to some relevant parts of the application.

**Review by roytomeij — APPROVED** [review]

**Comment by roytomeij** [comment]
From a tone of voice, etc point of view I think this is good to merge. Might need @matsimitsu for a technical check.

File: source/posts/logging/configuration.mdx [hunk 5]
```diff
@@ -50,93 +47,33 @@ Currently we only support sending logs to AppSignal using the Syslog message log
...
 
-Once you've created and saved your configuration, you will need to restart rsyslog so that it begins sending logs to AppSignal, using the restart command from a terminal with root access:
+Once you've created a new source, you'll need to configure your application to send logs to AppSignal. This method should be the same method you defined when creating your log source.
 
-`/etc/rc.d/init.d/systemctl restart rsyslogd`
+#### Supported Logging Platforms
 
-If you are unsure about this step, you can always [reach out](/support) for assistance. We'll help you to get those logs flowing!
+- [AWS Kinesis / CloudWatch](platforms/cloudwatch.html)
+- [Heroku Logdrian](platforms/heroku.html)
```
Comment (roytomeij) [inline source/posts/logging/configuration.mdx:74]:
```suggestion
- [Heroku Logdrain](platforms/heroku.html)
```

**Comment by matsimitsu** [comment]
There are a few inconsistencies, but we have a call in an hour to run through those :)

**Comment by roytomeij** [comment]
Is this ready to be merged, @noccor?

**Pull request merged**